README.md

```diff
@@ -35,27 +35,27 @@ brew tap mimoo/eureka && brew install eureka
 **1.** You are trying to send *Bob* the file `myfile.txt`.Start by encrypting the file via:
 
 ```
-eureka -encrypt -file myfile.txt
+eureka myfile.txt
 ```
 
-which will return a one-time 256-bit AES key and create a new `myfile.txt.encrypted` file:
+which will return a one-time 256-bit AES key and create a new `myfile.txt.enc` file:
```

Comment:
what's the motive behind shortening this? It's true that extensions tend to be small, but I'm thinking of someone who encrypted something a while ago and forgot about what this `.enc` file on his desktop is.

Reply:
borrowed from #17 comment.